pinot-api/src/main/java/org/apache/pinot/client/ConnectionFactory.java

```diff
@@ -18,13 +18,20 @@
  */
 package org.apache.pinot.client;
 
+import org.apache.yetus.audience.InterfaceAudience;
```

Comment:
Please use annotations from pinot-common.

Reply:
acknowledge

Reply:
We can remove this.